app/data/test_dates.json

```diff
@@ -127,10 +127,38 @@
                                     "display": {
                                         "properties": {}
                                     },
-                                    "id": "question",
+                                    "id": "3b7d43d1-8e0e-499d-928e-7cd78be8525e",
```

Comment:
Why did we remove a text based id and replace it with a GUID given that our approach going forward is to use descriptive ids?